mw-webapp/src/component/title/Title.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import styles from "src/component/title/Title.module.scss";
+
+interface TitleProps {
+  /**
+   * Heading level (h1, h2, h3, h4, h5, h6)
```

Comment:
not neccessary to duplicate types in comments

Reply:
done